modules/wolfgame.py

```diff
@@ -787,6 +795,189 @@ def fpinger(cli, nick, chan, rest):
     var.LAST_PING = None
     pinger(cli, nick, chan, rest)
 
+@cmd("pingif", "pingme", "pingat", pm=True)
+def altpinger(cli, nick, chan, rest):
+    """Pings you when the number of players reaches your preference."""
+    if not rest or (rest.isdigit() and int(rest) == 0):
+        if is_user_altpinged(nick)[0]:
+            msg = "Your ping preferences have been removed (was {0}).".format(is_user_altpinged(nick)[1])
+            if chan == nick:
+                pm(cli, nick, msg)
+            else:
+                cli.notice(nick, msg)
+            toggle_altpinged_status(nick, 0, is_user_altpinged(nick)[1])
+        elif not rest:
+            if chan == nick:
+                pm(cli, nick, "You need to enter a number.")
+            else:
+                cli.notice(nick, "You need to enter a number.")
+        else:
+            if chan == nick:
+                pm(cli, nick, "You do not have any preferences set.")
+            else:
+                cli.notice(nick, "You do not have any preferences set.")
+    elif rest.isdigit():
+        rest = int(rest)
+        if rest >= var.MAX_PLAYERS:
+            if chan == nick:
+                pm(cli, nick, "Please select a number between 0 and {0}.".format(var.MAX_PLAYERS-1))
+            else:
+                cli.notice(nick, "Please select a number between 0 and {0}.".format(var.MAX_PLAYERS-1))
+        elif is_user_altpinged(nick)[1] == rest:
+            if chan == nick:
+                pm(cli, nick, "Your ping preferences are already set to {0}.".format(rest))
+            else:
+                cli.notice(nick, "Your ping preferences are already set to {0}.".format(rest))
+        elif is_user_altpinged(nick)[0]:
+            msg = "Your ping preferences have been changed from {0} to {1}.".format(is_user_altpinged(nick)[1], rest)
+            if chan == nick:
+                pm(cli, nick, msg)
+            else:
+                cli.notice(nick, msg)
+            toggle_altpinged_status(nick, rest, is_user_altpinged(nick)[1])
+        else:
+            if chan == nick:
+                pm(cli, nick, "Your ping preferences have been set to {0}.".format(rest))
+            else:
+                cli.notice(nick, "Your ping preferences have been set to {0}.".format(rest))
+            toggle_altpinged_status(nick, rest)
+    else:
+        cli.notice(nick, "You need to enter a non-negative integer.")
+
+def is_user_altpinged(nick):
+    if nick in var.USERS.keys():
+        cloak = var.USERS[nick]["cloak"]
+        acc = var.USERS[nick]["account"]
+    else:
+        return False, None
+    if acc and acc != "*":
+        if acc in var.PING_IF_PREFS_ACCS.keys():
+            return True, var.PING_IF_PREFS_ACCS[acc]
+    elif not var.ACCOUNTS_ONLY and cloak in var.PING_IF_PREFS.keys():
+        return True, var.PING_IF_PREFS[cloak]
+    return False, None
+
+def toggle_altpinged_status(nick, value, old=None):
+    # nick should be in var.USERS; if not, let the error propagate
```

Comment:
nick won't be in var.USERS if it is a fake nick. Speaking of which, how are you handling those? You should probably have !pingif silently reject fake nicks instead of blowing up.